hack/import-restrictions.json

```diff
@@ -0,0 +1,9 @@
+[
+   {
+      "baseImportPath" : "pkg/image",
```

Comment:
can we place that as .import-restrictions in the respective directories. Then the OWNER/REVIEWER logic automatically applies.